Translation/README.md

```diff
@@ -6,6 +6,57 @@ Translation architecture shows below:
 
 ![architecture](./assets/img/translation_architecture.png)
 
+The Translation example is implemented using the component-level microservices defined in [GenAIComps](https://github.com/opea-project/GenAIComps). The flow chart below shows the information flow between different microservices for this example.
+
+```mermaid
+---
+config:
+  flowchart:
+    nodeSpacing: 400
+    rankSpacing: 100
+    curve: linear
+  themeVariables:
+    fontSize: 50px
+---
+flowchart LR
+    %% Colors %%
+    classDef blue fill:#ADD8E6,stroke:#ADD8E6,stroke-width:2px,fill-opacity:0.5
+    classDef orange fill:#FBAA60,stroke:#ADD8E6,stroke-width:2px,fill-opacity:0.5
+    classDef orchid fill:#C26DBC,stroke:#ADD8E6,stroke-width:2px,fill-opacity:0.5
+    classDef invisible fill:transparent,stroke:transparent;
+    style Translation-MegaService stroke:#000000
+
+    %% Subgraphs %%
+    subgraph Translation-MegaService["Translation MegaService "]
+        direction LR
+        LLM([LLM MicroService]):::blue
+    end
+    subgraph UserInterface[" User Interface "]
+        direction LR
+        a([User Input Query]):::orchid
+        UI([UI server<br>]):::orchid
+    end
+
+
+    LLM_gen{{LLM Service <br>}}
+    GW([Translation GateWay<br>]):::orange
+    NG([Nginx MicroService]):::blue
```

Comment:
Why nginx is in the front of the user input?

Reply:
I saw a nginx microservice as a proxy server in docker compose, and UI server depends it.
https://github.com/opea-project/GenAIExamples/blob/main/Translation/docker_compose/intel/hpu/gaudi/compose.yaml#L77

should I remove it or put it somewhere else in the workflow?

Reply:
updated them accordingly.